### DISCUSSION TITLE
Issue: Cursor moves from different position than its current one

### DISCUSSION THREAD

**Discussion by SanmayJoshi** [gsantner/markor#1744] [General] [answered]
My first time here. Don't know if this is the right place to report. 🙂

## Issue:

Expected: cursor moves from the position where it is

Actual: cursor moves from start of the doc

How to reproduce the issue:
1. Make sure that in settings -> edit mode, 'Start on bottom' isn't turned on.
2. create a new file with .md extension (haven't checked for other extensions)
3. paste some content (may use [lorem ipsum generator](https://www.lipsum.com/).)
4. enter view mode by tapping eye icon
5. enter edit mode
6. tap somewhere in the middle of doc
7. try moving the cursor by swiping on space bar (try moving it to left first since in my case it doesn't move left but only right).

**Comment by gsantner** [comment]
Markor version? If not 2.9.0 update to it.

Am 27. Juni 2022 19:51:45 MESZ schrieb Sanmay Joshi ***@***.***>:

**Reply by SanmayJoshi** [reply]
![Screenshot_20220628-145459~2](https://user-images.githubusercontent.com/26383381/176144467-947f6408-424e-491f-bd31-a2e85c5d0f27.png)

Yes, version is 2.9.0. Sorry for not mentioning it firsthand. 

Here's additional info:
- Phone is ASUS Zenfone Max Pro M1
- Using keyboard app OpenBoard.

**Reply by gsantner** [reply, marked as answer]
Does this still happen at the latest Markor version?
